locales/pdk.pot

```diff
@@ -806,64 +842,80 @@ msgstr ""
 msgid "Invalid JSON in metadata.json: %{msg}"
 msgstr ""
 
-#: ../lib/pdk/module/metadata.rb:114
+#: ../lib/pdk/module/metadata.rb:92
+msgid "Module metadata does not contain any requirements"
+msgstr ""
+
+#: ../lib/pdk/module/metadata.rb:93
+msgid "Module metadata does not contain a \"puppet\" requirement"
+msgstr ""
+
+#: ../lib/pdk/module/metadata.rb:94
+msgid "\"puppet\" requirement in module metadata does not specify a \"version_requirement\""
```

Comment:
@jbondpdx should this sentence start with "The", so that we aren't starting with a "?

Reply:
Yeah, I think that's a good idea.